recipe/meta.yaml

```diff
@@ -1,208 +1,215 @@
 
+  - url: {{ github }}  # [linux64 or aarch64]
+    sha256: a4b0965e7193515930542cc558d922e24c1732d8240efde3e622e84d6c4416fe  # [linux64 or aarch64]
+
+outputs:
   - name: cuquantum
-    version: {{ version }}
+    version: {{ ".".join(sdk_version[:3]) }}
     build:
-      number: {{ build_num }}
+      number: {{ sdk_build }}
     requirements:
       build:
       host:
       run:
-        - {{ pin_subpackage('custatevec', max_pin='x.x.x') }}
-        - {{ pin_subpackage('cutensornet', max_pin='x.x.x') }}
+        - {{ pin_subpackage("cudensitymat", max_pin="x.x.x") }}
+        - {{ pin_subpackage("cutensornet", max_pin="x.x.x") }}
+        - {{ pin_subpackage("custatevec", max_pin="x.x.x") }}
     test:
       commands:
-        - echo "ok!"
-
-  - name: custatevec
-    version: {{ cusv_version }}
+        - echo "cuquantum metapackage {{ sdk_version_string }}"
+  - name: cudensitymat
+    version: 0.0.5
     build:
-      number: {{ build_num }}
+      number: {{ sdk_build }}
+      string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ mpi_suffix }}"
       script:
-        - mkdir -p $PREFIX/include                                            # [linux]
-        - mv include/custatevec.h $PREFIX/include/                            # [linux]
-        - mkdir -p $PREFIX/lib                                                # [linux]
-        - mv lib/libcustatevec.so* $PREFIX/lib/                               # [linux]
+        - "${RECIPE_DIR}/cudensitymat/cudm-install"
       run_exports:
-        - {{ pin_subpackage('custatevec', max_pin='x') }}
-      ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        - {{ pin_subpackage("cudensitymat", max_pin="x") }} {{ mpi_prefix }}_*
+      ignore_run_exports:
+        - {{ compiler("cuda") }}
       missing_dso_whitelist:
-        - '*/libcublas.so*'    # [(cuda_compiler_version or "").startswith("11")]
-        - '*/libcublasLt.so*'  # [(cuda_compiler_version or "").startswith("11")]
-      post-link: post-link  # this is supported by conda-build, but undocumented
+        - '*/libcutensor.so*'  # [cuda_compiler_version in ("11.8",)]
+        - '*/libcublas.so*'    # [cuda_compiler_version in ("11.8",)]
+        - '*/libcublasLt.so*'  # [cuda_compiler_version in ("11.8",)]
+      post-link: post-link
     requirements:
       build:
         - {{ stdlib('c') }}
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("cuda") }}
       host:
-        - libcublas  # [(cuda_compiler_version or "").startswith("12")]
-        - cuda-version {{ cuda_compiler_version }}
+        - cutensor >=2.0.2,<3
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version in ("11.8", "12.0", "12.6")]
+        - libcublas  # [cuda_compiler_version in ("12.0", "12.6")]
+        - {{ mpi }}  # [mpi != "nompi"]
```

Comment:
You're probably missing a depdendency on `cuda-cudart-dev` according to the build logs.

Reply:
Could you please point me to where in the logs you are seeing this?

If you are refering to the following message, it will be displayed regardless of whether `cuda-cudart-dev` is present or not:
```
To enable CUDA support, UCX requires the CUDA Runtime library (libcudart).
The library can be installed with the appropriate command below:

* For CUDA 11, run:    conda install cudatoolkit cuda-version=11
* For CUDA 12, run:    conda install cuda-cudart cuda-version=12
```

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1109631&view=logs&j=d93976e3-6ed1-588b-ebf8-e8a19c2becc4&t=8f815395-247d-5523-e57f-dfa3d88394bc&l=1477

Reply:
Thanks for pointing to the output! I was indeed able to get rid of the warning by adding `host` dependency on `cuda-cudart-dev`. Since all but the python packages are just repackaged binaries, is this really a requirement?

Reply:
This is actually a library build error. All math libraries are supposed to statically link to cudart so as to support minor version compatibility. cuDensityMat does not meet this requirement and for some reason QA did not catch this.

Reply:
😮

Reply:
I will raise this issue with the cuquantum team. I expect it will be addressed in the next release.

For the time being, I specified `host` and `run` requirements on `cuda-cudart-dev` --- this should be sufficient, right?

Reply:
For CUDA 11, you'll need to use `cudatoolkit` which contains `libcuda.so`, because the new package layout was introduced starting CUDA 12.0. This should also help us all understand why the recipe was written in the way it was (we handled the old/new layouts in the same recipe).

Reply:
Yes, we carried that over from the prior recipe.